src/routes/blog-route/draft-route.jsx

```diff
@@ -12,7 +12,7 @@ export function DraftRoute() {
           <Header.Profile />
         </Header>
       </div>
-      <div className="relative float-right -top-10left-3/4 mr-32 mt-10 flex">
+      <div className="z-10 relative float-right -top-10left-3/4 mr-32 mt-10 flex">
         <Link to={"/draft"} className="underline">
           Drafts
```

Comment:
This solution seems to work great! 
But it seems like, just from a UX standpoint, the "Drafts" button on the /drafts route should probably change to say "Published" instead (people probably won't want to click on a button to see drafts if they are already on the drafts page). I'll add this to Tomer's queue to consider for a separate Jira story.

I'm calling this "Approved" because your solution totally matches the AC on the Jira story!

Reply:
I agree! It should say 'published' instead of 'drafts', and clicking it should route us to the published blogs section.